### DISCUSSION TITLE
RBAC session recording access

### DISCUSSION THREAD

**Discussion by scastromx** [gravitational/teleport#16147] [Q&A]
Just wondering if there is a way to restrict access to session recordings per role, for example to recordings of servers that has an specific label value.

Have tried with the "where" clause and cannot find the proper syntax.

**Comment by webvictim** [comment]
The `where` config doesn't currently allow this as far as I'm aware. You could try `where: contains(session.node_labels, "label-value")` but I'm not hopeful that it'd work.

The RFDs for the `where:` condition in the audit log are here:
- https://github.com/gravitational/teleport/blob/master/rfd/0044-session-where-condition.md
- https://github.com/gravitational/teleport/blob/master/rfd/0045-ssh_session-where-condition.md

You can also see some examples in the tests inside the codebase: https://github.com/gravitational/teleport/blob/0cdd93db0fb5e7ae866d16e244aaa06c18bfb171/lib/services/parser_test.go#L28-L155
